### DISCUSSION TITLE
Welcome to `A Cosmology Group` - Join Here!

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#15] [Announcements]
This is the public ACG forum on [`Discussions`](https://github.com/orgs/a-cosmology-group/discussions), please bookmark the link [https://github.com/orgs/a-cosmology-group/discussions](https://github.com/orgs/a-cosmology-group/discussions).

AI slop is not welcome on this forum.
AI can be used as a language tool, but ***not for content generation***.

### Discussions:

To post new messages and receive email notifications, you must:
- Create a personal [GitHub account](https://github.com/) at [https://github.com/](https://github.com/), and
- "Watch" the [`acg`](https://github.com/a-cosmology-group/acg) repository [https://github.com/a-cosmology-group/acg](https://github.com/a-cosmology-group/acg) by clicking the top-right button `Watch --> All Activity`.

To customize your email subscription, click (top-right)`Your Profile Picture` --> `⚙️Settings` --> (left column)`🔔 Notifications` and check or uncheck the appropriate boxes under **Subscriptions** and **System**.  See also [here](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications).

Equations can be written directly on the forum, GitHub renders equations written in LaTeX/MathJax (see [Writing mathematical expressions](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)).  This is useful for complex equations, like this one:

```
$$D_C(z) = \int_0^z \frac{{\rm d}z'}{\sqrt{\Omega_m(1+z')^3 + \Omega_\Lambda}}$$
```

which appears as

$$D_C(z) = \int_0^z \frac{{\rm d}z'}{\sqrt{\Omega_m(1+z')^3 + \Omega_\Lambda}}$$

For more details about the *Markdown syntax*, see [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

### Website:
The ACG website is deployed from the [`acg` repository](https://github.com/a-cosmology-group/acg) and visible at [cosmology.info](https://cosmology.info).

Members are encouraged to modify the [website repository](https://github.com/a-cosmology-group/acg) by [following these instructions](https://github.com/a-cosmology-group/acg/blob/main/assets/how_to/HOWTO.md).  This is where *an author can drop a piece of work, people can comment and all reviews stay with the work.*

`MathJax3` automatically translates LaTeX equations into `.html` webpages.  Webpages written in Markdown `.md` are encouraged, but `.html` and `.pdf` documents are also accepted.

**Comment by Francois-Zinserling** [comment]
If I clicked the "Watch --> All Activity" under repository, will I also get notification of all discussions, including new ones?

I did not click this before because 'Repository' seemed like a separate category, and something I might want to follow at a later stage. I think I might have misunderstood. 
Have clicked now but nobody is commenting so I can't see if it works! 🤣

**Reply by ftessier** [reply]
Yes, `Watch → All activity` will send you an email for all new discussions and comments, at the email address associated with your github account.

If you get this message by email, then it works! The best way to read discussions and participate is via this web address: https://github.com/orgs/a-cosmology-group/discussions, but you can also reply by email.

**Comment by Francois-Zinserling** [comment]
Replying by email … czech1,2

Note: I received the message below via email, but not your edited version.

Point I’m trying to make, I think some people (like me) might have thought joining was all that was required, and “Watch → All activity” was optional for repository only.

“Watch All” is crucial because this whole thing IS the repository. Not a Github user, so I didn’t know that!

Doing the “Watch All” makes it very similar to the email group we had before, and I don’t think there’s a problem with the Github solution.

Maybe it takes time for people to rejoin, but that would be the case with any new platform.

**Reply by ftessier** [reply]
Thank you for trying it out, and for the vote of confidence. It is very helpful to understand what the hurdle is, thank you for pointing out that the `Watch` setting seemed optional to you!

(Note that there is a link `View it on github` at the bottom of the email, to get to this discussion directly)

@marmetl, given this comment (and the preceding discussion), perhaps in the Fall one tack would be to ask members to send you their github handle (by email). You can then _invite_ them as collaborators on the repo: they would then only need to "accept", in which case the `Watch` setting is automatically set to `All activity`.

**Reply by budrap00** [reply]
Hi Frederic,

One of the problems with the github to email system is that it does not include in the email thread comments made on the same thread by you (the person receiving the email) in git hub. For instance a couple of weeks ago I received an email update to an existing thread consisting of a remark by Louis that was clearly commenting on something someone had said but I could not find the context in the email. When I went to github the context was obvious - he was commenting on a comment I had made - a comment that was not in the email thread. It is not unusual in ACG for comment responses to have a lag time of several days, so this behavior is going to be a further source of confusion for new users.

In Gaggle Mail, your comments were added to the email thread by GM whether you made them in email or using the mail tool. This resulted in a double entry when replying via email - annoying perhaps but less confusing than no entry at all when using the github mail tool. Hopefully this is just a forwarding-rule setting that can be switched off/on. Thanks for all your work on this.

Bud

**Reply by ftessier** [reply]
Thank you for this report and explanation, your are right. My perspective was biased then, because my habit is to click on the "View it on github" link at the bottom of the email. I presumed, wrongly, that others would do the same...

**Comment by RedshiftDrift** [comment]
*Paradigm Shifts* are difficult because people resist change.  The transition of ACG to GitHub is an *easy* paradigm shift that all ACG members should easily accept.  The assumption here is that *If you can't figure out GitHub, then you can't figure out the Cosmos.*

As @Francois-Zinserling said, *I don’t think there’s a problem with the Github solution.*

One motivation for the transition to GitHub is to enable users to post webpages with a summary of their thoughts and their papers, and improve these pages through discussions.  ACG members now have to upload their own articles themselves.  @clmarchesan posted [this webpage](https://github.com/a-cosmology-group/acg/blob/main/models/time-dependent-cosmologies/4seu/index.md), but is still waiting for comments.

@budrap00 
> extract the discussion page from the coder's environment it is embedded in

This is not possible, GitHub `Discussions` are all about the *code*.  From ACG's perspective, `Discussions` are all about cosmology as presented on the [website](https://cosmology.info/).

> the default setting for all viewers should be cosmology discussions only. Admin discussions should only be seen by those who want/need to see them

GitHub determines the default settings, but you can change your personal settings.  I expanded the instructions on how to do this in [this discussion](https://github.com/orgs/a-cosmology-group/discussions/15).  However, I would encourage receiving notifications about website modifications so you know when new article/improvements appear on the site.

> put a direct link to the discussions on the cosmology.info page

Good idea, I added the link to `Discussions` at the top-right of the webpage and in the left-column menu.


@ExpEarth 
> the 'feel' of the website might not have been as personal as before.

This is done on purpose, this is the AC***Group*** website.

> a computer code site, and loaded with jargon (I still don't know what a pull request is, for example), and there wasn't a clear roadmap to using the new site

The new site allows users to edit the webpage.  This is done with "pull requests" which are documented on hundreds of websites indexed by Google and modeled by ChatGPT.  See the [ACG wiki](https://github.com/a-cosmology-group/acg/wiki).

> Perhaps ACG could exist as a subgroup of Reddit

Reddit doesn't satisfy the group's requirement to render equations. @khuramonline inserting equations in a text is extremely inconvenient and time consuming.  Also, some users read their emails as 'character only' text, so they don't see the images.
The web version of GutHub discussions provides a standardized platform and renders all discussions in the same way for everybody.


@ftessier
> one way to channel attention to cosmology.info would be via the Cosmology Calculators...  I would extract them from the "Code" section, put a link on the home page, and ensure that they show up in search engine

Good idea, I'll replace `Code` with `Cosmology Calculators` in the website menu.

> one task would be to ask members to send you their github handle. You can then invite them as collaborators on the repo

More work for me and there is no point forcing people if they are not interested.  As Stacy McGaugh wrote "You can lead a theorist to data, but you can't make him think."

@ExpEarth 
> ...I can send to Roy and Craig Fraser and others.  We need to get Ray Fleming and Chantal back.

Yes, it would be good to have Roy, Craig, Ray, Chantal, etc. back.  I have done my part, but anyone can invite them again with the link [https://github.com/orgs/a-cosmology-group/discussions/15](https://github.com/orgs/a-cosmology-group/discussions/15).

**Comment by budrap00** [comment]
`The assumption here is that If you can't figure out GitHub, then you can't figure out the Cosmos.`

Louis, you should be ashamed of your self for even thinking that.

**Reply by Francois-Zinserling** [reply]
My thoughts were the same though, and I actually had a good giggle at that one. 🤣

**Comment by ExpEarth** [comment]
> Yes, it would be good to have Roy, Craig, Ray, Chantal, etc. back. I have done my part, but anyone can invite them again with the link https://github.com/orgs/a-cosmology-group/discussions/15.

I wonder about inviting new members though.  This page is not too inviting, a bit intimidating actually.  And if we were to invite someone using the link to the Discussions page, it is also not so clear what the new reader should do.  Instead of "Click here first", how about "Join"?  Then when they click on that they would see the instructions.  But these could still be improved in their general appearance.  Sorry to say, but a little more web design is needed!

Also, there seem to be a few issues regarding the ACG name.  First off, there is another ACG in GitHub, which is a computer group: (url) https://github.com/acgtic211 (url).  There is also the "Alternative Cosmology Group", the precursor to this group I think.  As well there is the Amsterdam Cosmology Group.  My point is that we might not be so easily visible on the web.

**Comment by luisenrique-arch** [comment]
I hope this is the right thread for this kind of suggestion.

I was thinking about something. I know this forum is dedicated to cosmology, and I think it's absolutely right that most discussions should stay focused on that. In fact, cosmology is the reason I joined in the first place, and it's the main reason I keep coming back.

That said, I think it might be nice to have a section for threads that are slightly off-topic. I don't mean off-topic in the sense of personal conversations or everyday life, but rather discussions about other areas of science. For example, topics like exobiology, geology, chemistry, atmospheric physics or other scientific disciplines that aren't directly related to cosmology.

Cosmology should, of course, remain the primary focus of the forum, but having a place for broader science discussions could make the community even more engaging.

What do you think?

**Reply by HanDeBruijn** [reply]
Here is the layout of a soccer field.

<img width="275" height="411" alt="550x821_HdB" src="https://github.com/user-attachments/assets/a9856b2d-ef79-4ff0-a4d4-5294d5b8239b" />

We already have incredibly much freedom on our ACG playfield.
But the field is limited by red lines.
To indicate that it is **game over** outside them.
A few such **offsite** positions are indicated with balls in red.
Sure, it happens more than once that a ball goes offsite.
What we must do then is throw the ball into the playfield again.
The worst thing we can do is to change the boundaries of ACG.
Because once such a process is started, there is no end to it.
(Now I could throw up bad examples in society, but I don't)

>it might be nice to have a section for threads that are slightly off-topic. 

Defninitely NO, sorry, I think it's an extremely bad idea.

**Reply by admin** [reply]
@luisenrique-arch thanks for your suggestion, but I will keep this forum focused exclusively on cosmology.  This was its goal when ACG was founded in 2004 (and initially hosted on *Yahoo!*)

However, the main reason for limiting discussions to the topic of cosmology  is  that the ACG forum needs constant attention to avoid an increasing entropy, being taken over by cranks, and then fall apart (like it did in 2010).  This maintenance requires reading the posts, managing requests, and keeping discussions under control (scientifically and socially :fearful:).

Doing all this takes time away from my research in cosmology, so the ACG forum must be worth my time by enabling an exchange of ideas with the scientific community involved with cosmology.

Other forums are available to discuss the topics you mention (like Quora, Reddit, an [instance of Mastodon](https://github.com/nathanlesage/academics-on-mastodon), etc.)  You can also follow YouTube channels and join their discussion group for a small fee (you already know about DemystifySci, [*See the Pattern*'s Gareth Samuel](https://www.youtube.com/@SeethePattern) is also interested in these topics), or [Facebook groups](https://www.facebook.com/groups/663488548027128/posts/980948956281084/), etc.

And if you don't like any of these, you can start your own on GitHub!

**Reply by luisenrique-arch** [reply]
@RedshiftDrift 

Thank you for your thoughtful reply. I completely understand where you're coming from. 

I only discovered this forum fairly recently. I didn't even know there was an earlier version of it that had to be shut down after descending into chaos. Had I known that, I never would have made this suggestion.

The main reason I made that suggestion is that I've found the people on this forum to be genuinely likeable, and I appreciate the open-minded attitude that many of you have toward different ideas. So I thought, "Hey, it could be fun to discuss other scientific topics with them, especially since they're already open to alternative cosmological models." But if doing that would end up creating the kinds of problems you described, then I'm happy to withdraw my suggestion.

**Reply by RedshiftDrift** [reply]
@luisenrique-arch  Thanks for the kind words about this group.  (Some "filtering" was applied to keep only the best people :wink:)

GitHub is not structured for private messaging, but most ACG members have a website or papers from which you could get their contact information.  Others remain anonymous.

Since you are already on GitHub, you could [create your own public repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories) (it's free) and [create your own Discussion forum](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/enabling-or-disabling-github-discussions-for-a-repository).  Then advertise your forum here to attract the people you like, and start building your group from there.

**Comment by ericcosmos** [comment]
At the top of this thread, @RedshiftDrift said "AI can be used as a language tool", and I wanted to take that idea for a spin. I thought, why not just feed the whole website into an AI and see what it says, especially since models have gotten noticeably better over the last few years. It actually came back with a pretty positive read. One thing it picked up on was that the newsletters feel like they’re written by an underdog that could actually pull it off. It also noticed your line, "ChatGPT won’t be writing the ACG Newsletter anytime soon!" from the 2023 newsletter (https://cosmology.info/newsletters/2023acg01newsletter.pdf) which was a nice catch. When the AI has enough context, it engages with the material instead of defaulting to the usual “Big Bang explains everything” framing. If you feed it all the ACG newsletters, it starts taking the ideas seriously.

I guess what I'm trying to say is that AI could end up being a really useful tool for taking apart the Big Bang narrative **as long as you give it the ACG newsletters to work with first**.

**Reply by luisenrique-arch** [reply]
@ericcosmos

I've noticed the same thing myself. If you discuss alternatives to the Big Bang with ChatGPT or Gemini without providing any context, they usually fall back on the standard talking points about how the Big Bang explains the observations, how well the Lambda-CDM model works, and how alternative models fail to account for the evidence. However, if you actually send them papers to analyze, they start engaging with alternative models much more seriously instead of just repeating what the consensus says.

A while ago, I uploaded David Crawford's paper on Curvature Cosmology to ChatGPT. It told me that the model was genuinely interesting, that the mathematical derivations were internally consistent and that the calculations appeared to be correct. I later did the same with the various pro-TL papers written by Nikita Lovyagin, Alexander Raikov, Vladimir Yershov and Yuri Lovyagin, and I got a very similar response. ChatGPT said the papers were well written, that the calculations were correct and that the criticisms they raised deserved to be taken seriously. I even sent it @RedshiftDrift's paper on Spectral-Transfer Redshift, and I received the same response: everything works.

So yes, if you give AI actual papers to analyze, it will provide thoughtful responses and take alternative models seriously.

**Reply by Francois-Zinserling** [reply]
I have had some really good results while working with AI. No, you can't just ask it to write a wall of text and equations and expect it to 'solve the universe', but you can gain a lot of ground by asking it targeted - focused - questions, and then interrogate it on the response.

Some time ago I started a website (ditched it already) where I took fringe papers on aether and ask AI to score it. This is where I learnt what kind of questions to ask to get consistent results. E.g. you may ask AI to review GR it will tell you it's the best thing ever. Then ask it a question about MOND and dark matter and dark energy and the cosmological constant ... and then ask it about GR again and, well, suddenly GR is not so great after all. <- Just an example I made up. If you find AI starts 'drifting', start a new chat window asap.

ChatGPT: I found it tried too much to flatter the author/reader by telling you how well you are doing and how you did such a great job, while it's not actually working. With science you need brutal responses. 
Gemini: Worked very well, except soon (every time) started using the document I gave it as a 'source of information', instead of continuously checking it against other models.
DeepSeek: My clear favourite by far. Yes, it can lose the plot entirely sometime, but it has geven me valuable insight even when I wasn't asking for specifics. This is what you want for exploratory models. It also tends to get 'sold' on the model after a while in the chat window so caution is advised.

On all of these it's useful to do quick complex equations, instead of doing it myself, it's much easier to check if AI got it right. 9/10 they do but that 1/10 will bury you if you don't spot it.

These were all unpaid versions. Not sure if anyone had better responses from other LLMs.

**Reply by HanDeBruijn** [reply]
A few remarks from my side.
- Neural Networks are at the foundation of Artificial Intelligence.
- I started to read a voluminous book about it and produced a [viXra article](https://vixra.org/abs/2605.0027).
  It barely scratches the surface though (: what do you expect)-:
- Our pope Leo XIV has written an [Encyclical Letter](https://www.vatican.va/content/leo-xiv/en/encyclicals/documents/20260515-magnifica-humanitas.html) about AI. Worthwile.

As about the above comment and replies: _AI is truly an **Intelligence**_. With other words: intelligence is not limited to humans. It has been stated already in the 1969 (!) [book by Stefan Denaerde](https://github.com/orgs/a-cosmology-group/discussions/225#discussioncomment-10019679): _dat intelligentie, geheugen en normbesef **materi&euml;le zaken** zijn (that intelligence, memory, and a sense of norms are **material things**)_. That's the reason why machines can be intelligent. Next question: what makes **us** different from a machine then?

**Reply by HanDeBruijn** [reply]
@Francois-Zinserling .
>These were all unpaid versions. Not sure if anyone had better responses from other LLMs.

A paid version is e.g. Claude Opus. I have been told by someone in the know that it shows far better responses than free LLMs. And I have seen what it produces when fed with my own homepage on cosmology, but decided not to publish yet.

**Reply by RedshiftDrift** [reply]
> the newsletters feel like they’re written by an underdog that could actually pull it off - @ericcosmos 

I learned a lot writing these newsletters.  In addition, I reviewed papers I think I could explain with STz, so this made the newsletter consistent over the years.

> yes, if you give AI actual papers to analyze, it will provide thoughtful responses and take alternative models seriously. - @luisenrique-arch 

That's a problem though; AI does try "[too much to flatter the author](https://github.com/orgs/a-cosmology-group/discussions/15#discussioncomment-17480657)" and can be made to agree with anything.  All the papers you fed the AI cannot be right since some contradict others, but AI can't figure which one (if any) is right.

> that 1/10 will bury you if you don't spot it - @Francois-Zinserling 

I did try free AI for a derivation of the CMB temperature using only first principles and STz.  Guess what it gave me?  2.7 K!
Then I found that it had done a bad job for the derivative of a polynomial, had cherry picked terms that would give 2.7 K as a final result, and had given nonsensical arguments to support that answer.  Without guidance, AI is a risky endeavour.

@luisenrique-arch I asked the opposite question about my paper: "What would mainstream cosmologists say about my paper?"  Answer: my paper is all wrong :rofl: :rofl:

> started using the document I gave it as a 'source of information' - @Francois-Zinserling 

All LLMs do that, even the paid ones.  That's why the AI will agree with you, because you've trained it with your own ideas by giving it your paper!

>  what makes us different from a machine then? - @HanDeBruijn 

As opposed to AI, we are not trained to regurgitate what other people have written.  Humans generate genuinely new ideas.

**Reply by luisenrique-arch** [reply]
@RedshiftDrift 

You’re right, AI is designed to agree with whoever it’s talking to. However, there are settings that can try to tone this down. For example, on ChatGPT I’ve completely disabled the option to always agree with me, and in fact, every time I talk about cosmology with ChatGPT, it’s practically impossible to make it change its mind unless I bring it actual research papers. The same goes for other topics where I hold positions that aren’t aligned with the mainstream.

In any case, I don’t ever use AI for scientific topics, unless it’s about things where there’s universal consensus, like the size of the Earth, the size of the continents, the composition of the atmosphere, the distances between the Sun (or the Earth) and the other planets, etc. I usually just use AI to translate texts from Italian (my native language) into English.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift writes:
>As opposed to AI, we are not trained to regurgitate what other people have written.

Oh well, I think that many (if not all) of us **have** been trained to regurgitate what other people have written.
It’s called **supervised learning**, very much the same process as with the training of neural networks in AI.
>Humans generate genuinely new ideas.

Yes, that is a missing link, I think: CREATIVITY !!

**Reply by RedshiftDrift** [reply]
> I think that many (if not all) of us have been trained to regurgitate what other people have written. - @HanDeBruijn 

Not me, I've always had the suspicion that people are trying to impose their beliefs for control, enhance their career, or to make money.  E.g. the bible, Santa Claus, spirituality vs science (remember "The Tao of Physics"?), statistics (like "Thirteen against the bank" which claimed you could win at roulette!), philosophy proving the existence of God, history showing that all British people were bad because the invaded Québec (where I'm from), the "point particle" in quantum mechanics, etc. etc.  All nonsense.

Then I read about the Big Bang model with the same suspicion: someone is pushing an idea to enhance their career.

Yes, we were trained in school to behave like AI, but I think people on this forum ***must*** do better!
Hence my dislike of AI and rejection of posts that regurgitate what others have written.

**Reply by ftessier** [reply]
What is interesting in AI is the transformer neural network architecture (on which LLMs are largely based on today). It reveals that "predicting what comes next" is a key aspect of what we call "intelligence". This is why language has for a long time been considered as a loose proxy for "intelligence" and "thought" in general (see Wittgenstein, for the most flamboyant example!).

Language is better viewed not as the ability to utter words or communicate, but more comprehensively as the machinery for **"predicting what comes next".** AI is not simply regurgitating what exists (which presuppose replicate encoding, absent in LLMs), it is building an inference engine. This is what biological brains do! Creativity is not a different operation (inference), but a question of degree (infer beyond the existing corpus).

**Reply by ftessier** [reply]
Great progress in science has hinged on humans removing themselves or their narrow perspective from some self-declared importance or self-built pedestal. This is the well-known [Copernician principle](https://en.wikipedia.org/wiki/Copernican_principle) (heuristic of course): **_assume you're not special, and the science gets better._** Modern AI is that same move, for what we call _intelligence_.

Examples:

- Earth is not the center of the Universe, neither is the sun or the galaxy  (astronomy, cosmology).
- Humans as just another species (evolution).
- Life isn't chemically special, nor is Earth (chemistry).
- Mind isn't a special substance (psychology, neuroscience).
- _"We are more of an environment than an organism"_ 😉, microbiome symbiosis (biology, physiology).
- No special frame of reference (relativity).
- No absolute physical state, only entangled observations (quantum mechanics).

I am not saying that any of the above is settled or true, they should continue to be probed and questioned. But outgrowing the human perspective seems to be a genuinely productive attitude in science. Another wonderful promise in the same vein — and of interest to this community — is _removing the very human conception of time and space_ (even relative!) from fundamental physics (removal of time arguably gestured at by the [Wheeler-DeWitt](https://en.wikipedia.org/wiki/Wheeler%E2%80%93DeWitt_equation) equation).

**Reply by HanDeBruijn** [reply]
@ftessier writes:
>**assume you're not special, and the science gets better.**

That's where my heart emoi is for.
>But outgrowing the human perspective seems to be a genuinely productive attitude in science. 

Having noticed the extreme individualism in this group, I would like to propose the following extension.
_But outgrowing the **my own model** perspective seems to be a genuinely productive attitude in science._
>Another wonderful promise [ .. ] is _removing the very human conception of time and space_ (even relative!) from fundamental physics.

That's where my confused emoi is for.
The concept of space-time instead of [**space and time**](https://github.com/orgs/a-cosmology-group/discussions/344) is the worst thing that has ever happened to physics.
Even worse is the completely incomprehensible [Wheeler-DeWitt](https://en.wikipedia.org/wiki/Wheeler%E2%80%93DeWitt_equation) equation. Following [Sabine Hossenfelder](https://www.youtube.com/watch?v=shFUDPqVmTg), I call such stuff **BS** physics.

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn the concept of space-time as an intertwined entity is not as foreign if you consider the posiibility of an aether. Even Einstein acknowledged in ~1920 that we require an aether for physics to work. By that time SR and QM had gained such momentum without an aether, coupled to the fallacy that Michelson and Morley disproved the aether, that Einstein's comment gained no traction.

Let's say hypothetically we are emerged in a Planckian aether, with Planck length 'photons' isotropic and each has length = $$L_P$$. The 'photons' pass a length $$L_P$$ in a time $$t_P$$ which naturally leads to the speed of light  $$c=L_P/t_P$$. (you can verify this relation)
If nothing can move faster than the aether then that sets the universal speed limit, $$c$$. You could start by inserting $$c$$ and then getting $$t_P$$ or $$L_P$$ but these are all constants now so the relation is set - for an isotropic aether.
With a discrete length unit of $$L_P$$ and a universal ticking rate of $$t_P$$ the foundation for space-time is set. These are inseparable unless you can alter the speed of light. 

Incidentally we could, e.g. in a gravitational well c is lower and clocks tick slower. Then there exists a metric:

$$ds^2=-f(x)c^2dt^2+{f(x)}^{-1}dx^2$$

where f(x) is the rate of time flow (change) due to the presence of mass. f(x) is how mass curves space.
In GR, to first order:

$$f(x)=1-2GM/{c^2r}$$

Long story short. We do not know what space is nor do we know what time is, and space-time seems an unnecessary complication, but it is actually a simplification of both.

**Reply by luisenrique-arch** [reply]
@Francois-Zinserling I've said it before, I'll say it again: I think Lorentz was on the right track. Treating space-time as a purely geometrical entity only leads to absurd complications and nonsensical paradoxes. Physics needs an aether and a preferred frame of reference. It simply doesn't work without both.

**Reply by Francois-Zinserling** [reply]
> Having noticed the extreme individualism in this group,...

This is a sad state of affairs, especially for a group like this. Our only common ground appears to be 'NOT λCDM' but there is no confinement to be measured in 'everything else'. 

Still, that is a good common ground to have and I highly value the input and criticisms of all in this group.

**Reply by ftessier** [reply]
I support efforts to derive physics in a way where "time" and "space" are emergent properties rather than axioms of the theory. Anytime we write "t" or "x",  ill-defined anthropocentric concepts, we are making a sweeping assumption that Nature can fundamentally be described in the same way our brains are interpreting it. The "t" and "x" variables have been useful up to now, manifestly, yet they betray a singular and limiting world-view. My wager is that once we find how to build physics without these concepts baked in, much of physics, including cosmology, will leap forward.

**Reply by HanDeBruijn** [reply]
>a length $L_P$ in a time $t_P$ which naturally leads to the speed of light $c=L_P/t_P$.

Trivially so, @Francois-Zinserling , because $L_P$ and $t_P$ are _defined_ in this way. Ipse est: circular reasoning.

>Then there exists a metric:

There exist no other metric than the Euclidean one for space: $ds^2 = dx^2 + dy^2 + dz^2$.
And there exist two separate time frames: [Atomic time and Orbital time](https://github.com/orgs/a-cosmology-group/discussions/344), $t$ and $T$, where

$$
\left(1+\frac{T}{A}\right) = \exp\left(\frac{t}{A}\right) \quad \mbox{with} \quad A=\mbox{"Age"}
$$

With some name-changing and $\large A=\frac{1}{H}$ the [Time Dilated Past hypothesis](https://github.com/orgs/a-cosmology-group/discussions/160) by @mikehelland is recovered.

>We do not know what space is nor do we know what time is

Denied. We **do** know very precisely how Eclidean Geometry works for empty space.
And we **do** know how Atomic clocks and Orbital clocks accurately measure time.

@luisenrique-arch .

>Physics needs an aether and a preferred frame of reference.

Physics does NOT need a single preferred frame of reference. [Fictitious forces](https://github.com/orgs/a-cosmology-group/discussions/215#discussioncomment-9475066) are not fictitious , they are **real**.

@ftessier .

>Anytime we write "t" or "x", ill-defined anthropocentric concepts

How about "m" for mass or "T" for temperature, aren't these _ill-defined anthropocentric concepts_ too?

**Reply by ftessier** [reply]
Of course they are ill-defined too! That mass is emergent is already well-accepted in traditional physics, and T is ill-defined both because 1) technically, the Boltzmann constant $k_\text{B}$ is not covariant; 2) its definition leans heavily on entropy, which is _perspectival_ (and incidentally our only link to time in physics, and in our thermodynamic brains...).

**Reply by ftessier** [reply]
We ought to lift the foregoing discussion(s) to separate discussion threads, outside this "Welcome to ACG!" one.

**Comment by ftessier** [comment]
Continuing the "emergent time and space" discussion in #383.

**Reply by HanDeBruijn** [reply]
@ftessier .
A similar discussion does already exist:
[space, time and philosophical considerations from an experimentalist's point of view](https://github.com/orgs/a-cosmology-group/discussions/215).
Wouldn't it be a better idea to join forces there, instead of creating more of the same?
(: nobody seems to be interested)

**Comment by RedshiftDrift** [comment]
AI is not all bad, [this article](https://phys.org/news/2026-07-ai-qled-recipe-efficiency-boosts.html) reports how AI tried a large number of combinations to improve the efficiency of a QLED.  The researchers actually built the QLED found by AI and confirmed the increased efficiency.

On this discussion forum, AI will be accepted if its output is *built* and *works better*, e.g.
- the translated text reads better, or
- you have built the universe proposed by AI and confirmed that it works better than the observed universe.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .
>- you have built the universe proposed by AI and confirmed that it works better than the observed universe.

Sorry ?! How ever can a universe be built, in theory, that _works better than the observed universe_ ?

>AI will be accepted if [ .. ]

Here in the Netherlands we say: "_Als je ze een vinger geeft, dan pakken ze je hele hand_".
Okay, let Google Translate (AI?) be my friend: _Give them an inch and they'll take a mile_.

**Reply by RedshiftDrift** [reply]
> Sorry ?! How ever can a universe be built... - @HanDeBruijn 

It can't be done, therefore no AI theory will be accepted on ACG Discussion!

As for giving 'een vinger', you can guess which one I give to AI :rofl:
